Ginger/GingerCoreNET/Run/GingerRunner.cs

```diff
@@ -994,6 +994,11 @@ public void RunAction(Act act, bool checkIfActionAllowedToRun = true, bool moveT
             catch(Exception ex)
             {
                 Reporter.ToLog(eLogLevel.ERROR, "Exception in Run Action", ex);
+                if (string.IsNullOrEmpty(act.Error))
+                {
+                    act.Error = "Exception in Run Action " + ex;
+                    act.Status = eRunStatus.Failed;
```

Comment:
We should not set Status explicitly. At the end of executiion there is function which caluclates actions status. If act.Error has some value it will mark it as failed